src/app/course/course-homepage/course-homepage.component.scss

```diff
@@ -0,0 +1,75 @@
+@import '~@taiga-ui/core/styles/taiga-ui-local';
+@import 'mixins';
+
+.course-tile {
+    border: 0.2rem solid #000000;
+    overflow: hidden;
+    cursor: default;
+    @include screen-lg {
+        flex-basis: calc(30% - 4.5rem);
+    }
+    @include screen-md {
+        flex-basis: calc(50% - 4.5rem);
+    }
+    @include screen-sm {
+        flex-basis: 100%
+    }
+    margin: 1rem;
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: column;
+}
+
+.flex-container {
+    margin: auto;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    flex-direction: row;
+    @include screen-lg {
+        max-width: 1200px;
+    }
+    @include screen-md {
+        max-width: 900px;
+    }
+    position: relative;
+}
+
+.flex-space {
+    flex-grow: 3;
+    width: 95%;
```

Comment:
95% ?!

Reply:
Without this text was occasionally overflowing, I will see if I can find a better fix

Reply:
I think this should be resolved now